lib/lolcommits/cli/ui.rb

```diff
@@ -0,0 +1,168 @@
+require 'methadone'
+require 'clamp'
+
+# allow logging from all commands
+# TODO: we should now rationalize using Methadone for this vs something for just logging
+include Methadone::CLILogging
+
+module Lolcommits
+  module CLI
+
+    class EnableCommand < Clamp::Command
+      def execute
+        # TODO: rationalize how to pass options to Installation.enable
+        # previous version relied on all flags being global (yikes)
```

Comment:
The options to `Installation.do_enable(options)` end up just being a string, they are only used to populate the hook script with some default capture_args [here](https://github.com/mroth/lolcommits/blob/eef879aad2f25f4cd296b03982c724eed0c5e3c3/lib/lolcommits/backends/installation_git.rb#L63) ...  so maybe you can get them from `ARGV` directly?